.gitignore

```diff
@@ -8,3 +8,5 @@ node_modules/
 vendor/
 venv/
 **/.DS_Store
+# locally we don't need mise to manage ruby. netlify does
+mise.local.toml
```

Comment:
this isn't needed, but since we're using rvm to manage the local ruby install i created the file ☝️ to ensure that mise ignores it entirely in the DevContainer and that Netlify continues to use mise to install it.

```
[settings]
idiomatic_version_file_enable_tools = ["node"]
```

if you don't do this. its harmless, a call to `node -v` just triggers mise to install both backends.